### PR TITLE
Schwartz cmds

### PR DESCRIPTION
**Pull request by Schwartz10**
@jimpick can you help me wrap this one up? Can we add a `--from` flag for `transfer-ownership` and `transfer-operator` so that the person can specify the new owner or operator? Also, can we make sure that we properly update the address cache with the new addresses (presumably only in agent.toml)

### CODE REVIEW

**Comment by jimpick** [comment]
I'll add support for a `wallet-resolver.toml` with mappings from symbolic names such as "owner" and "operator" to addresses (EVM for now, but possibly Filecoin later).

File: cmd/utils.go [hunk 1]
```diff
@@ -85,13 +86,28 @@ func ParseAddressToNative(ctx context.Context, addr string) (address.Address, er
...
+		if err != nil {
+			return common.Address{}, err
+		}
+		defer closer()
+
+		return parseAddress(ctx, addr, lapi)
+	}
+
+	as := util.AccountsStore()
+	evmAddr, _, err := as.GetAddrs(addr)
```
Comment (Schwartz10) [inline cmd/utils.go:106]: Why did this logic get added here?

File: cmd/utils.go [hunk 2]
```diff
@@ -192,27 +201,33 @@ func commonOwnerOrOperatorSetup(ctx context.Context, from string) (agentAddr com
...
 		return common.Address{}, nil, accounts.Account{}, nil, err
 	}
 
 	var fromAddress common.Address
 	// if no flag was passed, we just use the operator address by default
-	switch from {
-	case "", opEvm.String(), opFevm.String():
-		funded, err := as.IsFunded(ctx, PoolsSDK, opFevm, util.OperatorKeyFunded, opEvm.String())
+	switch strings.ToLower(from) {
+	case "", opEvm.String(), opFevm.String(), string(util.OperatorKey):
```
Comment (Schwartz10) [inline cmd/utils.go:229]: Does the `strings.ToLower(from)` interfere with potentially checksummed addresses resulting from the `.String()` method?

I.e. if `opEvm.String()` returns checksummed address, then this case will never match?

File: cmd/utils.go [hunk 3]
```diff
@@ -453,3 +524,20 @@ func checkUnencryptedPrivateKeys() error {
...
+	defer closer()
+
+	bal, err := lapi.WalletBalance(ctx, caller)
+	if err != nil {
+		return false, err
+	}
+	if bal.Cmp(big.NewInt(0)) > 0 {
+		return true, nil
+	}
+	return false, nil
```
Comment (Schwartz10) [inline cmd/utils.go:542]: Not important but you can also just:

```
return bal.Cmp(big.NewInt(0)) > 0, nil
```

File: cmd/utils.go (hunk 1 above)
Comment (jimpick) [reply cmd/utils.go:106]: This function was already accepting either an eth or a fil address, and if it was a fil address, it was going a lookup for the id address using Lotus.

This was used by a lot of the CLI commands that accept addresses, and I needed to add in the ability to also accept account names (eg. 'default', 'owner', 'operator').

The name of the function should probably be renamed because it now accepts both account names and addresses.

File: cmd/utils.go (hunk 2 above)
Comment (jimpick) [reply cmd/utils.go:229]: It definitely does. This is a bug.

I'll wrap the `.String()` methods to make them lowercase too.

Comment (jimpick) [reply cmd/utils.go]: Appears to be fixed.

File: cmd/utils.go (hunk 1 above)
Comment (jimpick) [reply cmd/utils.go:106]: I renamed it to `AddressOrAccountNameToEVM()` ... a bit wordy, but I'm bad at naming things.

Comment (Schwartz10) [reply cmd/utils.go:106]: Got it! That's cool - so you should be able to transfer or withdraw to named accounts as well
